.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "mocha": true,
+        "jasmine": true
+    },
+    "plugins": [
+        "html"
```

Comment:
It sure is.

ESLint has a convention, where a npm module named "eslint-plugin-foo" can be used in an ESLint config file with:

```js
"plugins": [
    "foo"
]
```